engine/crates/engine-v2/src/request/bind.rs

```diff
@@ -173,6 +178,7 @@ pub struct Binder<'a> {
     // We also need to use the global request position so that merged selection sets are still
     // in the right order.
     next_response_position: usize,
+    current_fragments_stack: VecDeque<String>,
```

Comment:
If I'm reading the file right, it looks like we're only pushing and popping at the back, so a `Vec<String>` should work too?

Reply:
good point, you're right